pkgs/top-level/all-packages.nix

```diff
@@ -28691,12 +28691,13 @@ with pkgs;
     mpris = callPackage ../applications/video/mpv/scripts/mpris.nix {};
     mpv-playlistmanager = callPackage ../applications/video/mpv/scripts/mpv-playlistmanager.nix {};
     mpvacious = callPackage ../applications/video/mpv/scripts/mpvacious.nix {};
+    quality-menu = callPackage ../applications/video/mpv/scripts/quality-menu.nix { };
     simple-mpv-webui = callPackage ../applications/video/mpv/scripts/simple-mpv-webui.nix {};
     sponsorblock = callPackage ../applications/video/mpv/scripts/sponsorblock.nix {};
     thumbnail = callPackage ../applications/video/mpv/scripts/thumbnail.nix { };
     vr-reversal = callPackage ../applications/video/mpv/scripts/vr-reversal.nix {};
-    youtube-quality = callPackage ../applications/video/mpv/scripts/youtube-quality.nix { };
     cutter = callPackage ../applications/video/mpv/scripts/cutter.nix { };
+    youtube-quality = throw "youtube-quality is no longer maintained, use quality-menu instead"; # added 2022-12-26
```

Comment:
This should go into `pkgs/top-level/aliases.nix`.

Reply:
How can I add this to `aliases.nix` without causing `error: Alias mpvScripts is still in all-packages.nix`?